test/basic/init_funcs.c

```diff
@@ -0,0 +1,16 @@
+#include "smack.h"
+
+// @expect verified
+
+void __SMACK_init_func_test() {
```

Comment:
Wait, should:

void __SMACK_INIT(test)(){
...
}

be expanded to 

void __SMACK_init_func() {
...
}

or 

void __SMACK_init_funct_test() {
...
}

?

I assume you mean the latter?